docs/standards/universal-profile/lsp6-key-manager.md

```diff
@@ -890,6 +890,22 @@ Since the Key Manager offers **relay execution** via signed message, it's import
 
 - Signed messages with sequential nonces should be **executed in order**, meaning a signed message with nonce 4 can't be executed before the signed message with nonce 3. This is a critical problem which can limit the usage of relay execution.
```

Comment:
There is a typo above:

**exist** ✅ , not **existed** ❌ 

<img width="328" alt="image" src="https://github.com/lukso-network/docs/assets/31145285/46b8ad7e-d6a8-45de-b3bb-82dd1ea485c9">